src/modules/Elsa.Persistence.EntityFrameworkCore/Modules/WorkflowSink/Configurations.cs

```diff
@@ -0,0 +1,21 @@
+using Elsa.Workflows.Sink.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Elsa.Persistence.EntityFrameworkCore.Modules.WorkflowSink
+{
+    public class Configurations :
+        IEntityTypeConfiguration<WorkflowSinkEntity>
```

Comment:
I think the term "Workflow Sink Entity" is confusing, because it's not intuitive what it means. What it should mean, is a view of the workflow state being stored in the database. So perhaps a better name would be e.g. "WorkflowInstance", which is more accurate.

So perhaps better would be to move the current EF core code surrounding the WorkflowInstance entity to the WorkflowSinks folder in this project and remove WorkflowSinkEntity.